basis/html/templates/chloe/chloe-tests.factor

```diff
@@ -10,7 +10,7 @@ IN: html.templates.chloe.tests
 
 : run-template ( quot -- string )
     with-string-writer [ "\r\n\t" member? ] reject
-    "?>" split1 nip ; inline
+    "<!DOCTYPE html>" split1 nip ; inline
```

Comment:
A few of our templates seem to have DOCTYPES that are a little different:

```
$ find . -iname "*.xml" | xargs grep DOCTYPE
./extra/webapps/help/search.xml:<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
./extra/webapps/mason/download-package.xml:<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
./extra/webapps/mason/download-release.xml:<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
./extra/websites/concatenative/page.xml:<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
```

They either have to get fixed, or change the splitting logic...?